tests/performance_tests/performance_test_basic.py

```diff
@@ -21,26 +21,144 @@
 relaunchTimeout = 30
 emptyBlockGoal = 5
 
-def waitForEmptyBlocks(node1):
+class blockData():
+    def __init__(self):
+        self.transactions = 0
+        self.cpu = 0
+        self.net = 0
+        self.elapsed = 0
+        self.time = 0
+        self.latency = 0
+
+class chainData():
+    def __init__(self):
+        self.blockLog = []
+    def sum_transactions(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.transactions
+        return total
+    def sum_cpu(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.cpu
+        return total
+    def sum_net(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.net
+        return total
+    def sum_elapsed(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.elapsed
+        return total
+    def sum_time(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.time
+        return total
+    def sum_latency(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.latency
+        return total
+    def print_stats(self):
```

Comment:
Add a `__str__(self)` method which calls this method.  Then `chainsContainer` can simply `print(self.preData)` and `print(self.totalData)` as needed.